gradle-plugin/src/main/java/com/squareup/anvil/plugin/DisableIncrementalCompilationTask.kt

```diff
@@ -23,7 +23,7 @@ internal abstract class DisableIncrementalCompilationTask : DefaultTask() {
 
```

Comment:
Is this Gradle Task still needed? Looks like https://youtrack.jetbrains.com/issue/KT-38570 is fixed now :)

Reply:
I need to find time to test it. I filed #627.